<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficCaseInformationDialog</class>
 <widget class="QDialog" name="TrafficCaseInformationDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>867</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Case Information</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>961</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="fines_amount"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Offense:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Plea:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="offense_choice_box">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="court_costs_box">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Yes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Finding:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="plea_choice_box">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Not Guilty</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No Contest</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Guilty</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No Plea - Dismissed</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Court Costs:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="statute_choice_box">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="fines_suspended"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Fines Suspended:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="finding_choice_box">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Guilty</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Not Guilty</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dismissed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dismissed with conditions</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Statute / Degree:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Fines Amount:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="addOffenseButton">
       <property name="text">
        <string>Add Offense</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="degree_choice_box">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>MM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UCM</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="deleteOffenseButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Delete Offense</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_4">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Clear Fields</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>961</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>260</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="charges_gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="offense_label_1">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Offense:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Statute:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Fines Suspended:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="plea_label_1">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Plea:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Court Costs:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Finding:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Fines:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Degree:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>961</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>941</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,2,0">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="plea_trial_date">
       <property name="minimumDate">
        <date>
         <year>2021</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2021</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Case Number:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="case_number">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Defendant Name: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="defendant_name">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Operator License Number:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="defendant_attorney_name">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Date of Birth:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1980</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Clear Fields</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>961</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Defendant shall pay fines and costs:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="ability_to_pay_box">
       <item>
        <property name="text">
         <string>forthwith</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>within 30 days</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>within 60 days</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>within 90 days</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Balance of fines and costs due by:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="balance_due_date">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2021</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="createEntryButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>830</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Entry</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>case_number</tabstop>
  <tabstop>defendant_name</tabstop>
  <tabstop>defendant_attorney_name</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>offense_choice_box</tabstop>
  <tabstop>statute_choice_box</tabstop>
  <tabstop>plea_choice_box</tabstop>
  <tabstop>finding_choice_box</tabstop>
  <tabstop>fines_amount</tabstop>
  <tabstop>fines_suspended</tabstop>
  <tabstop>court_costs_box</tabstop>
  <tabstop>ability_to_pay_box</tabstop>
  <tabstop>plea_trial_date</tabstop>
  <tabstop>createEntryButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>defendant_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>defendant_attorney_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>809</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>case_number</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addOffenseButton</sender>
   <signal>clicked()</signal>
   <receiver>TrafficCaseInformationDialog</receiver>
   <slot>add_offense()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>959</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>980</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createEntryButton</sender>
   <signal>released()</signal>
   <receiver>TrafficCaseInformationDialog</receiver>
   <slot>close_window()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>908</x>
     <y>838</y>
    </hint>
    <hint type="destinationlabel">
     <x>927</x>
     <y>787</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createEntryButton</sender>
   <signal>clicked()</signal>
   <receiver>TrafficCaseInformationDialog</receiver>
   <slot>create_entry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>837</x>
     <y>838</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>784</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createEntryButton</sender>
   <signal>pressed()</signal>
   <receiver>TrafficCaseInformationDialog</receiver>
   <slot>update_case_information()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>877</x>
     <y>822</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>740</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addOffenseButton</sender>
   <signal>clicked()</signal>
   <receiver>offense_choice_box</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>959</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addOffenseButton</sender>
   <signal>clicked()</signal>
   <receiver>plea_choice_box</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>959</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addOffenseButton</sender>
   <signal>clicked()</signal>
   <receiver>fines_amount</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>959</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addOffenseButton</sender>
   <signal>clicked()</signal>
   <receiver>statute_choice_box</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>959</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>847</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addOffenseButton</sender>
   <signal>clicked()</signal>
   <receiver>finding_choice_box</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>959</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>847</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addOffenseButton</sender>
   <signal>clicked()</signal>
   <receiver>fines_suspended</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>959</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>847</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offense_choice_box</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>TrafficCaseInformationDialog</receiver>
   <slot>set_statute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ability_to_pay_box</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>TrafficCaseInformationDialog</receiver>
   <slot>set_pay_date()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>753</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>835</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>create_entry()</slot>
  <slot>continue_dialog()</slot>
  <slot>close_window()</slot>
  <slot>set_dialog()</slot>
  <slot>update_case_information()</slot>
  <slot>ovi_dialog()</slot>
  <slot>sentencing_dialog()</slot>
  <slot>proceed_to_sentencing()</slot>
  <slot>proceed_to_ovi()</slot>
  <slot>add_offense()</slot>
  <slot>set_statute()</slot>
  <slot>set_pay_date()</slot>
 </slots>
</ui>
